### DISCUSSION TITLE
[Scam Alert] Marsmasters

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#23] [General]
Marsmasters
ادرس سایت https://usdtpool.marsmasters.com/
براین ادعاست که با استیک کردن تتر سود بگیرید .

![302](https://user-images.githubusercontent.com/103103225/184469101-67daed44-0969-4014-9fdb-56f0228efa2a.jpg)

بررسی ها :
ارائه بسته های سرمایه گذاری های با سود اغواگرانه ::
پکیج اول ماهی 17.5 درصد سود با حداق سرمایه گذاری 5000 دلار و دوم ماهی 70 درصد سود با حداقل سرمایه گذاری 1000 دلار

![312](https://user-images.githubusercontent.com/103103225/184469113-03f32011-d920-4969-adcd-bf6ed857e5f6.jpg)

در سایت، لینک پشتیبانی تلگرام قرارداده شده است که باورود به اون میبینیم کاملا ایرانی است  در کانال میبینیم به سایت هایی اشاره شده که بررسی میکنیم
https://t.me/MarsMastersSupport

![300](https://user-images.githubusercontent.com/103103225/184469119-0e672c8a-86f8-4bc3-8242-cd37de1d34b8.jpg)
![301](https://user-images.githubusercontent.com/103103225/184469120-0ee26852-0d12-448f-98a1-64591fba3f21.jpg)

حال وارد این سایت یعنی marsmasters.com میشویم  که با ارور زیر مواجه شدیم . ولی با اطلاعات whois سایت 2019 ساخته و 2022 اپدیت شده است اطلاعات با ادرس پرووایدر معروفی که خیلی از سایت های اسکم استفاده میکنند مخفی شده است.

![303](https://user-images.githubusercontent.com/103103225/184469127-ca884e44-402b-457c-91b2-668f93ea976f.jpg)
![311](https://user-images.githubusercontent.com/103103225/184469130-92b04d05-4a04-46cb-a98c-719725847627.jpg)

در کانال از فردی به نام اقای محسن فرمنش به عنوان مدیرعامل کمپانی مارکو نام برده شده است 

![305](https://user-images.githubusercontent.com/103103225/184469131-8ce74fad-8680-4704-b589-cece708c813c.jpg)

با تحقیقاتی در سایت https://bitboy.academy/teacher/mohsen-farmanesh/ اطلاعات کامل وی مشخص شده است که اکادمی بیت بوی از ایشان است .

![318](https://user-images.githubusercontent.com/103103225/184469141-3207ba39-4fad-41cd-82ca-4e552238a7b3.jpg)

اطلاعات در مورد توکن و پلتفرم مارکو :

![306](https://user-images.githubusercontent.com/103103225/184469148-0a6c65c2-10e3-4180-8860-13fb548dd919.jpg)

سایت رسمی پلتفرم غیر متمرکز مارکو:
www.Marco.Travel 
آدرس Smart Contract:
0xdc3228e10259494A834743260CA8340c7Ea90391
در ضمن طبق تحقیق ادرس کانترکت توکن مارکو x5c792740a7d4684E75e5551Fb239f512Fe0d47d00 است Marco Token (MRC)
که حدود 62 درصد توکن ها در دست 4 ولت است  و ساخته سال 2021

![320](https://user-images.githubusercontent.com/103103225/184469157-15dc9759-3812-4251-8c9e-8eeb9ecfc948.jpg)

وضعیت سایت با Marco.Travel فیلترشکن و بدون فیلترشکن.

![307](https://user-images.githubusercontent.com/103103225/184469162-45ede182-2f28-4abe-951b-613973bc6372.jpg)

و وجود سایتی دیگر در کانال تلگرامی pool.marsmasters.com 
که این سایت هم در مورد استیک کردن کوین ها و توکن هاست همچنین دارای پلن هایی از سود های نامتعاف مخصوصا با استیک کردن توکن مارکو 

![317](https://user-images.githubusercontent.com/103103225/184469165-4ad5ed60-d1e2-4c61-b325-430eea394fc2.jpg)

که 
1.  از سایت های  mrcpool.marsmasters.com برای استیک توکن ماکرو 
![313](https://user-images.githubusercontent.com/103103225/184469174-b3d3c45f-52db-43d1-84ca-288c2ca1ff2f.jpg)

 3. از سایت btcpool.marsmasters.com برای استیک بیت کوین 

![314](https://user-images.githubusercontent.com/103103225/184469183-2be75ebc-93fc-492d-b079-3cad911b0372.jpg)

5. از سایت usdtpool.marsmasters.com برای استیک تتر 

![315](https://user-images.githubusercontent.com/103103225/184469186-5407280d-b58e-4af9-8708-71c86d3ec574.jpg)

7.از سایت ethpool.marsmasters.com برای استیک اتریوم 

![316](https://user-images.githubusercontent.com/103103225/184469192-56a13220-b6d3-4217-b189-804b53fb811e.jpg)

8. همچنین به همین ترتیب برای xtz  و busd  از سایت هایی استفاده میکنند . 

و در ادامه رونمایی از توکنی دیگر https://starsx.io/ که این هم با ادعای خودشان ساخته خودشان است. 

![309](https://user-images.githubusercontent.com/103103225/184469200-b68bfbee-87b2-4acb-bf92-1d1ab3f67351.jpg)
![310](https://user-images.githubusercontent.com/103103225/184469202-a8067d07-1c9f-4018-9802-6f0a473984bd.jpg)
![319](https://user-images.githubusercontent.com/103103225/184469203-6e1e4319-c80f-49e1-9a0f-25fb1173ac80.jpg)

و در نهایت ولت واریزی usdt  این شبکه که تمامی واریزی ها بجای استیک در ولت فرد از ولت اصلی خارج شده است . 

![321](https://user-images.githubusercontent.com/103103225/184469207-6fc9c8dd-2cc1-49d3-a813-9615ebfc7833.jpg)

هشدار اسکم و کلاهبرداری

**Comment by Blockunity** [comment]
![IMG_20220813_125508_370.jpg](https://user-images.githubusercontent.com/72666334/184480513-2ee36c8a-c294-46f6-95da-0f8685afae54.jpg)
![IMG_20220813_125506_415.jpg](https://user-images.githubusercontent.com/72666334/184480516-085acf66-0dab-4db0-9fe7-11f6df15e2e8.jpg)
 
ولت پروژه با 1.5 میلیون دلار واریزی تا الان  TAHjsyyYkmzXQWRqRPTcaNQwCbf4TC9xk9 
ولت احتمالی دزد اصلی پروژه با 57 میلیون دلار واریز  
TJ1rruZp13DCnUbMTiqeCDUmN413HXTKBv
از تاریخ ها مشخصه که چند پروژه موازی با هم بالا آورده و وردی و خروجی هاش از یک تاریخ شروع شده .
